pdf/model/structures.go

```diff
@@ -124,6 +125,25 @@ func NewPdfDate(dateStr string) (PdfDate, error) {
 	return d, nil
 }
 
+func NewPdfDateFromTime(timeObj time.Time) (PdfDate, error) {
```

Comment:
Could also be written as:

```
func NewPdfDateFromTime(timeObj time.Time) (PdfDate, error) {
	timezone := timeObj.Format("-07:00")
	utOffsetHours, _ := strconv.ParseInt(timezone[1:3], 10, 32)
	utOffsetMins, _ := strconv.ParseInt(timezone[4:6], 10, 32)

	return PdfDate{
		year:          int64(timeObj.Year()),
		month:         int64(timeObj.Month()),
		day:           int64(timeObj.Day()),
		hour:          int64(timeObj.Hour()),
		minute:        int64(timeObj.Minute()),
		second:        int64(timeObj.Second()),
		utOffsetSign:  timezone[0],
		utOffsetHours: utOffsetHours,
		utOffsetMins:  utOffsetMins,
	}, nil
}
```

Reply:
https://github.com/unidoc/unidoc/wiki/UniDoc-Developer-Guide#12-doc-comments